CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (gRPC) [#23844](https://github.com/cosmos/cosmos-sdk/pull/23844) Add debug log prints for each gRPC request.
```

Comment:
i think this should rather go under `improvements`. wdyt?

Reply:
fixed ty!

Reply:
[Merge branch 'release/v0.53.x' into chore/495e185e2deb8200f817ddae39c…](https://github.com/cosmos/cosmos-sdk/pull/23844/commits/d89d13c567d3161c3e4b20908e0f7e68c2e085dc)